### ISSUE TITLE
video.tv-tokyo.co.jp

### ISSUE THREAD

**Issue by Yuki2718** [uBlockOrigin/uAssets#7515]
### URL(s) where the issue occurs

`https://video.tv-tokyo.co.jp/shinmai/episode/00075162.html`

### Describe the issue

Annoyance: again questionnaire

### Screenshot(s)

![tv-tokyo](https://user-images.githubusercontent.com/58900598/83863281-19cf4580-a75e-11ea-8fb6-db5dfa7d45e4.png)

### Versions

- Browser/version: Brave 1.9.76
- uBlock Origin version: 1.27.10

### Settings

- Default + Fanboy Annoyances + uBlock Annoyances

### Notes

Geo-restricted

**Comment by mapx-** [comment]
Does this work ?
`tv-tokyo.co.jp##+js(nostif, /^/)`

**Comment by Yuki2718** [comment]
> 
> 
> Does this work ?
> `tv-tokyo.co.jp##+js(nostif, /^/)`

No, broke the video.

**Comment by mapx-** [comment]
```
video.tv-tokyo.co.jp##.md-question.js-modal-window
video.tv-tokyo.co.jp###jsiModalOverLayerShare
```

**Comment by mapx-** [comment]
@ryanbr  for Fanboy's annoyances list

**Comment by Yuki2718** [comment]
I forgot to mention `@@||pubads.g.doubleclick.net/gampad/ads?$xmlhttprequest,domain=tver.jp|video.tv-tokyo.co.jp` is also needed for this too (reported in EL forum but not yet fixed).

@mapx- 
The fix is incomplete, the video will be overshadowed soon after start.

![tv-tokyo](https://user-images.githubusercontent.com/58900598/83942802-e100a080-a831-11ea-98e8-6bfaf92dada6.png)

I got the same result when I blocked `cdn.ic.tver.jp`.
